diffgraph/utils/git_utils.py

```
@@ -0,0 +1,106 @@
+    dangerous_flags = {
+        '--name', '--name-only', '--name-status', '--pretty', '--numstat',
+        '--format', '--exec', '--output-format', '--color',
+        '--no-color', '--color=always', '--color=auto', '--color=never'
+    }
+
+    # Safe flags that are commonly needed
+    safe_flags = {
+        '-U', '--unified', '-R', '--reverse', '-B', '--break-rewrites',
+        '-M', '--find-renames', '-C', '--find-copies', '--find-copies-harder',
+        '-D', '--irreversible-delete', '-l', '--max-count', '-S', '--find-object',
+        '-G', '--pickaxe-regex', '--pickaxe-all', '--pickaxe-regex',
```

Ruff (0.12.2)

Sets should not contain duplicate item `'--pickaxe-regex'`

Remove duplicate item

(B033)